project-docs/wave-01.md

```diff
@@ -4,9 +4,15 @@
 
 Update the `ChatEntry` and `App` components to display a single chat message bubble with the message text and relative timestamp, plus the sender's name above it.  
 
-A good way to test this code as you write it would be to take the first chat message from the JSON data file and use it as the data for the `ChatEntry` component.
+A good way to test this code as you write it would be to take the first chat message from the [JSON data file](src/data/messages.json) and use it as the data for the `ChatEntry` component.
```

Comment:
Needs the path relative to the project-docs folder
```suggestion
A good way to test this code as you write it would be to take the first chat message from the [JSON data file](../src/data/messages.json) and use it as the data for the `ChatEntry` component.
```